extensions/sort-mentions/src/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { Clipboard, Toast, getPreferenceValues, getSelectedText, showToast } from "@raycast/api";
+import { exec } from "child_process";
+
+export const handleInput = async (handle_type: Preferences["primary_handle"] | Preferences["secondary_handle"]) => {
+  const selectedText = await getSelectedText();
```

Comment:
If not selecting any text it crashes, could you handle this gracefully? You can see an [example here](https://developers.raycast.com/api-reference/environment#example-3)

![sort-mentions 2023-07-25 at 09 42 58](https://github.com/raycast/extensions/assets/38033107/0f34b560-7901-4493-aab0-0e059a73e641)